frontend/assets/l10n/app_de.json

```diff
@@ -5,6 +5,10 @@
     "disclaimer": "Haftung, Haftungsausschluss und Impressum",
     "infoTitle": "Info",
     "languageChange": "Sprache wechseln",
+    "languages(map)": {
+      "en": "Englisch",
+      "de": "Deutsch"
+    },
```

Comment:
I think the languages in a language switch menu are usually not translated but only shown in their mother tongue.

Reply:
ok so you think we should use the `deviceLocale` ?

Reply:
I propose to always show each language in its own language :D
So "Deutsch", "English", (... "Français", ...) etc, just like we do it in the integreat app.

Reply:
Indeed and therefore no translations are needed

Reply:
My main issue with the previous solution was that it was `Englisch`. Completely fine with using own language instead of translations.